packages/kernel-5.10/config-bottlerocket-metal

```diff
@@ -7,32 +7,67 @@ CONFIG_SATA_AHCI=y
 CONFIG_ATA=y
 CONFIG_ATA_PIIX=y
 
-# Intel network support
-CONFIG_IGB=m
-CONFIG_IGBVF=m
-CONFIG_NET_VENDOR_INTEL=y
-CONFIG_IGB_HWMON=y
-CONFIG_E1000=m
-CONFIG_E1000E=m
-CONFIG_E1000E_HWTS=y
+# AMD network support
+CONFIG_NET_VENDOR_AMD=y
+CONFIG_AMD_XGBE=m
+# CONFIG_AMD_XGBE_DCB is not set
 
 # Broadcom network support
 CONFIG_NET_VENDOR_BROADCOM=y
 CONFIG_TIGON3_HWMON=y
 CONFIG_TIGON3=m
+CONFIG_BNX2X=m
+CONFIG_BNX2X_SRIOV=y
 CONFIG_BNXT=m
 
+# Chelsio network support
+CONFIG_NET_VENDOR_CHELSIO=y
+CONFIG_CHELSIO_T4=m
+CONFIG_CHELSIO_T4VF=m
+# CONFIG_CHELSIO_T4_DCB is not set
+# CONFIG_CHELSIO_INLINE_CRYPTO is not set
+# CONFIG_INFINIBAND_CXGB4 is not set
+# CONFIG_ISCSI_TARGET_CXGB4 is not set
+
 # Cisco UCS network support
 CONFIG_NET_VENDOR_CISCO=y
 CONFIG_ENIC=m
 CONFIG_INFINIBAND_USNIC=m
 
+# Emulex network support
+CONFIG_NET_VENDOR_EMULEX=y
+CONFIG_BE2NET=m
+CONFIG_BE2NET_BE2=y
+CONFIG_BE2NET_BE3=y
+CONFIG_BE2NET_HWMON=y
+CONFIG_BE2NET_LANCER=y
+CONFIG_BE2NET_SKYHAWK=y
+
+# Huawei network support
+CONFIG_NET_VENDOR_HUAWEI
```

Comment:
This is just mentioning the config option, not giving any value (i.e. `=y`). Same in the config for kernel-5.15.

Reply:
Ouch, this is embarrassing... I thought I had worked out all these kinks. Fixed in the force push below.